src/contexts/tokens.context.tsx

```diff
@@ -32,8 +32,8 @@ interface ComputeWrappedTokenAddressParams {
 }
 
 interface GetNativeTokenInfoParams {
+  address: string;
```

Comment:
I would still pass the domain interface as we are doing in the bridge instead of just the address

Reply:
We can't because [we still don't have the token built](https://github.com/0xPolygonHermez/zkevm-bridge-ui/pull/241/files#diff-9c5cb3f673be0ba52c8570a1ccd23fcfab6bfc5ee91735b11a42d160c140f6e0R201).